src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageActionInvoker.cs

```diff
@@ -21,7 +21,7 @@
 
 namespace Microsoft.AspNetCore.Mvc.RazorPages.Internal
 {
-    public class PageActionInvoker : ResourceInvoker, IActionInvoker
+    internal class PageActionInvoker : ResourceInvoker, IActionInvoker
```

Comment:
Again, empty out the `.Internal` namespaces

Reply:
Focusing on `Mvc.Core` for now. Will do this in follow up